### PR TITLE
feat: add validate-file script

### LINKED ISSUE #57: Rework integration testing in sketch-file-format to account for new Sketch schema smoke test
At the moment sketch-file-format tests against a specific version of Sketch, but with the new testing strat the file format no longer needs knowledge of a specific Sketch version.

It might be useful however, to retain some sort of script in the file format repo that enables testing the schemas against an arbitrary Sketch file, for use during development.

### PR DESCRIPTION
**Pull request by jedrichards**
Resolves #57

### CODE REVIEW

File: src/validate-file.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        meta: JSON.parse(archive.entryDataSync('meta.json').toString()),
+        user: JSON.parse(archive.entryDataSync('user.json').toString()),
+      })
+    })
+  })
+
+  archive.close()
+
+  return contents
+}
```
Comment (christianklotz) [inline src/validate-file.ts]: Would `fromFile` be better placed somewhere else for others to use?

Comment (jedrichards) [reply src/validate-file.ts]: Definitely - but not sure it belongs here with the schemas 🤔

Comment (christianklotz) [reply src/validate-file.ts]: Maybe add a `TODO: …` for revisiting later?

Comment (jedrichards) [reply src/validate-file.ts]: Yeap.

**Pull request merged**